packages/flutter_tools/lib/src/cache.dart

```diff
@@ -1058,6 +1059,36 @@ class FlutterRunnerSDKArtifacts extends CachedArtifact {
   }
 }
 
+/// The debug symbols for flutter runner for Fuchsia development.
+class FlutterRunnerDebugSymbols extends CachedArtifact {
```

Comment:
This needs a test - there should be some existing examples in cache.dart

Reply:
cache_test.dart

Reply:
I've added a test. It doesn't actually test the downloading part, but tests the package resolution. I think this should cover any common failures.